provisioning/miniprov/hare_mp/main.py

```diff
@@ -1009,6 +1031,57 @@ def main():
         help='Target directory; defaults to /tmp/hare.',
         action='store')
 
+    sb_sub_parser.add_argument(
+        '--duration',
+        type=str,
+        nargs=1,
+        help='Time bound log collection, like logs for last n days /'
+             ' weeks / months',
+        action='store')
+
+    sb_sub_parser.add_argument(
+        '--size_limit',
+        type=str,
+        nargs=1,
+        help='Limited sized logs',
+        action='store')
+
+    sb_sub_parser.add_argument(
+        '--services',
+        type=str,
+        nargs=1,
+        help='With in component, logs collection limiting to a single'
```

Comment:
1. `s/With in/Within/g`
2. I think the expected syntax of the argument needs to be described.

Reply:
Addressed.